src/Editor/HoverProvider.ts

```diff
@@ -22,6 +22,8 @@ export class HoverProvider implements vscode.HoverProvider {
           {getWordRangeAtPosition: (arg0: any, arg1: RegExp) => any; getText: (arg0: any) => any;},
       _position: any, _token: any) {
     let mdfile = new vscode.MarkdownString();
+    const range = _doc.getWordRangeAtPosition(_position, new RegExp(/(.+)/g));
```

Comment:
is `range` always valid (that is not `null` nor `undefined`) ?

Reply:
I just checked that the range was undefined when I moused on the line where only the enter was entered.
Should I revise this part?

Reply:
For example, there is a way to perform the logic below when range is not undefined.